Tools/msi/bundle/Default.wxl

```diff
@@ -86,7 +86,7 @@ Select Customize to review current options.</String>
   <String Id="ShortPrependPathLabel">Add &amp;Python [ShortVersion] to PATH</String>
   <String Id="InstallAllUsersLabel">Install for &amp;all users</String>
   <String Id="InstallLauncherAllUsersLabel">for &amp;all users (requires elevation)</String>
-  <String Id="ShortInstallLauncherAllUsersLabel">Install &amp;launcher for all users (recommended)</String>
+  <String Id="ShortInstallLauncherAllUsersLabel">Install py.exe &amp;launcher for all users (recommended)</String>
```

Comment:
Bikeshedding: When customizing the installation, on the optional features page, it's called "py launcher", so I'd prefer it without the ".exe" extension, unless `Include_launcherLabel` is changed to `"py.ex &amp;launcher"`.